resources/views/settings/customization.blade.php

```diff
@@ -0,0 +1,93 @@
+@extends('settings.layout')
+
+@section('settings_content')
+    <h2 class="mb-4 h5">{{__('Customization')}}</h2>
+
+    <form method="post" action="{{route('settings.customization.update')}}">
+        @csrf
+        @method('PATCH')
+
+        <div class="row">
+            <div class="col-lg-6">
+
+
+                <x-input.text label="{{(__('Primary Color'))}}"
```

Comment:
instead of stating the default, make it not required and save the default back, or when it's empty use the default on the locations